content/blog/2020-04-14-devguide-release/index.md

```diff
@@ -0,0 +1,150 @@
+  - Karthik Ram
+  - Noam Ross
+  - Scott Chamberlain
+  - Melina Vidoni
+date: 2020-04-14
+categories: blog
+topicid:
+tags:
+  - Software Peer Review
+  - dev guide
+description: "Updates in version 0.4.0 of the online book 'rOpenSci Packages: Development, Maintenance, and Peer Review
+rOpenSci Packages: Development, Maintenance, and Peer Review'"
+twitterImg: img/blog-images/2019-05-16-dev-guide-update/cover.png
+---
+
+rOpenSci Software Peer Review's guidance has been compiled in [an online book](https://devguide.ropensci.org/) for more than one year now. We've just released its fourth version. 
+To find out what's new in our dev guide 0.4.0, you can [read the changelog](https://devguide.ropensci.org/booknews.html), 
+or this blog post for more digested information.
+
+> Note that, as indicated in the [README of the software review repo](https://github.com/ropensci/software-review), in the interest of reducing load on reviewers and editors as we manage the COVID-19 crisis, rOpenSci is temporarily pausing new submissions for software peer review for 30 days (and possibly longer). Please check back [in the software review repository](https://github.com/ropensci/software-review) again after 17 April for updates. In this period new submissions will not be handled, nor new reviewers assigned. Reviews and responses to reviews will be handled on a ‘best effort’ basis, but no follow-up reminders will be sent.
+
+### Policy and governance changes
+
+Some aspects of the software review process changed.
+
+#### Policy changes
+
+We've added field and laboratory reproducibility tools as a category in [scope](https://devguide.ropensci.org/policies.html#aims-and-scope). 
+_Packages that improve reproducibility of real-world workflows through standardization and automation of field and lab protocols, such as sample tracking and tagging, form and data sheet generation, interfacing with laboratory equipment or information systems, and executing experimental designs. (Example: [baRcodeR](https://docs.ropensci.org/baRcodeR/))_
+
+{{< tweet 1183811515067043840 >}}
+
+Our [reviewer guide](https://devguide.ropensci.org/reviewerguide.html) now includes guidance for off-thread interactions. We're glad for community building but don't want any important information to get lost.
+_If you interact with the package authors and talked about the review outside a review thread (in chats, DMs, in-person, issues in the project repository), please make sure that your review captures and/or links to elements from these conversations that are relevant to the process._
+
+Another change linked to policy is that we clarified our advice related to rOpenSci's Code of Conduct.
+That code of conduct applies to rOpenSci packages so repositories shouldn't use another Code of Conduct, and the [default `CODE_OF_CONDUCT.md` of the ropensci GitHub organization](https://github.com/ropensci/.github/blob/master/CODE_OF_CONDUCT.md) will be picked up by GitHub in e.g. new issues.
```

Comment:
I'm a little confused by what is meant here. If I understand correctly, the rOpenSci code of conduct applies automatically to all repositories so maintainers shouldn't include one? This way it's standardized and will pop up when users open issues etc. I also think that ropensci can be written as rOpenSci because that's the formatting of the organization name on it's page. 

Also the link to the CODE_OF_CONDUCT.md, isn't actually a markdown file with the code of conduct... should it be?

What do you think of the following suggestion?

```suggestion
Another change linked to policy is that we clarified our advice related to rOpenSci's Code of Conduct.
This Code of Conduct applies automatically to all rOpenSci repositories so maintainers should not include a Code of Conduct to avoid conflict. This way the [default `CODE_OF_CONDUCT.md` of the rOpenSci GitHub organization](https://github.com/ropensci/.github/blob/master/CODE_OF_CONDUCT.md) will be picked up by GitHub in e.g. new issues.
```